source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -191,22 +194,23 @@ and call the ``createCollection()`` method as shown in the following example:
 
 You can change the pre-image and post-image option in an existing collection
 by running the ``collMod`` command from the MongoDB Shell. To learn how to
-perform this operation, see the :manual:`collMod </reference/command/collMod#change-streams-with-document-pre--and-post-images>`
-{+mdb-server+} manual.
+perform this operation, see the entry on :manual:`collMod </reference/command/collMod/>`
+in the Server manual.
 
 .. warning::
 
    When you modify this option on a collection, any change streams open on
-   that collection in your application may fail if configured to require
-   receiving the pre-image or post-image.
+   that collection in your application may fail if you configure them to require
+   the pre-image or post-image.
```

Comment:
S: Might be able to trim this down a bit. [Style Guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#s:~:text=than%20matrices.-,may,-Use%20may%20to) also recommends using "may" only for permission

Maybe something like: 
"If you enable this option while change streams are open in your application, 
those change streams might fail."

Reply:
reworded